resources/css/style.css

```diff
@@ -71,7 +71,7 @@
 .filter-projects-inputs {
   display: flex;
   flex-flow: row wrap;
-  justify-content: space-evenly;
+  justify-content: space-evenly;  /* Ignore CSSLintBear */
```

Comment:
csslint doesn't have a supporting rule for this. It doesn't accepts `space-evenly` 
Already raised an issue related to this in upstream